Dockerfile

```diff
@@ -26,10 +26,18 @@ RUN apt-get update && \
   chmod 755 /run.sh
 
 # Volumes
-VOLUME /var/lib/unifi-video /var/log/unifi-video
+VOLUME ["/var/lib/unifi-video"]
+VOLUME ["/var/log/unifi-video"]
+VOLUME ["/usr/lib/unifi-video/data/videos"]
```

Comment:
Why is this needed? Wouldn't the first line include any sub-directories which would include `["/usr/lib/unifi-video/data/videos"]`

Reply:
Hi,
Because my Doker don't show the current works dirs... and I read in a Docker docs thats correctly is with this format....